doc-independent-rosindex.yaml

```diff
@@ -0,0 +1,20 @@
+%YAML 1.1
+# ROS buildfarm doc-build file
+---
+documentation_type: external_site
+doc_repositories:
+- https://github.com/ros2/rosindex.git
+jenkins_job_priority: 90
+jenkins_job_timeout: 120
+targets:
```

Comment:
Since this job runs mostly in a container from the repository source we should either remove the target stanza, or if that's not easily doable comment that it is ignored for this documentation type.

Reply:
I guess we could change the `BuildFile` base class definition to make that stanza optional on a case by case basis. I wonder though, do `make_target` doc_independent jobs make use of it?